bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/OneWireBinding.java

```diff
@@ -112,7 +112,7 @@ public void updated(Dictionary<String, ?> pvConfig) throws ConfigurationExceptio
 
     @Override
     protected void internalReceiveCommand(String pvItemName, Command pvCommand) {
-        logger.debug("received command " + pvCommand.toString() + " for item " + pvItemName);
+        logger.debug("received command {} for item {}", pvCommand.toString(), pvItemName);
```

Comment:
The `.toString()` isn't necessary (it costs *almost* nothing to do, but better to leave it to debug method in case it's not needed at all).